perf/terraform/modules/long_lived/files/user-data.sh

```diff
@@ -7,6 +7,14 @@ sudo yum -y install iperf3
 # Bump UDP receive buffer size. See https://github.com/quic-go/quic-go/wiki/UDP-Receive-Buffer-Size.
 sudo sysctl -w net.core.rmem_max=2500000
 
+# Set maximum TCP send and receive window to bandwidth-delay-product.
+#
+# With a bandwidth of 25 Gbit/s per machine and a ping of 60 ms between the two
+# machines, the bandwidth-delay-product is ~178.81 MiB. Set send and receive
+# window to 200 MiB.
+sudo sysctl -w net.ipv4.tcp_rmem='4096 131072 200000000'
+sudo sysctl -w net.ipv4.tcp_wmem='4096 20480 200000000'
```

Comment:
Do I understand correctly that this is set for every run, not just for iperf?

Reply:
Correct. This set on startup on both machines and is thus set for all benchmarks, all implementations and all versions.